File: Web Api/MascotApp-WS/MascotApp-WS/Models/Usuarios.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace MascotApp_WS.Models
{
    public class Usuarios
    {
        public string Id { get; set; }
        public string Usuario { get; set; }
        public string Email { get; set; }
        public string Contrasena { get; set; }
        public string Token { get; set; }
        public string Foto_Perfil { get; set; }
        public string Fecha_Creacion { get; set; }
        public string Rol { get; set; }
        public string Estado { get; set; }
    }
}

File: Android/MascotApp/app/src/main/java/co/com/bucompany/mascotapp/Entidades/Usuarios.kt
package co.com.bucompany.mascotapp.Entidades

data class Usuarios (
        val Id : String,
        val Usuario : String,
        val Email : String,
        val Contrasena : String,
        val Token : String,
        val Foto_Perfil : String,
        val Fecha_Creacion : String,
        val Rol : String,
        val Estado : String
)